conf.yaml

```
@@ -1430,6 +1430,21 @@ contents:
     -
         title:      Enterprise Search
         sections:
+          -
+            title:      Enterprise Search Guide
+            prefix:     en/enterprise-search
+            index:      workplace-search-docs/index.asciidoc
+            private:    1
+            current:    master
+            branches:   [ master ]
+            live:       *stacklive
+            chunk:      1
+            tags:       Enterprise Search/Reference
+            subject:    Enterprise Search
+            sources:
+              -
+                repo:   enterprise-search-pubs
+                path:   enterprise-search-docs
```

This path needs to align with the index field and point to a path that exists in the repo (there is no `enterprise-search-docs` folder that I can see):
```suggestion
                path:   workplace-search-docs
```

---

As mentioned @lcawl, the folder has not been merged to master yet. Workplace Search and Enterprise Search are two different books.